Show kubeless code (#2800)
kyma-project/kyma: components/kubeless-images/README.md

# Kubeless Images

## Overview

> This component creates runtime images, that would be used by the kubeless controller to instantiate lambdas. Currently we are supporting only nodejs for versions `node6` and `node8`.

Reviewer: ```suggestion
This component creates runtime images, to be used by the Kubeless controller to instantiate lambda functions. Currently, we are supporting only `node6` and `node8` versions of NodeJS.
```
